src/openms/source/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderIdentificationAlgorithm.cpp

```diff
@@ -313,7 +314,7 @@ namespace OpenMS
         PeptideHit seed_hit;
         seed_hit.setCharge(f_it->getCharge());
 
-        const String pseudo_mod_name = String(100000 + seeds_added);
+        const String pseudo_mod_name = String(1);
```

Comment:
I still need to check if this works.

Reply:
can you confirm this works. e.g., on a larger dataset with potential a lot of stuff  eluting we still get same number of quantified features if seeds are used?